schema/gen/src/lib.rs

```diff
@@ -453,10 +549,8 @@ mod tests {
 
     #[test]
     fn no_extra_or_missing_schemas() {
-        let exceptions: HashSet<_> = RawGenesisBlock::schema()
-            .into_iter()
-            .map(|(type_id, _)| type_id)
-            .collect();
+        // NOTE: Skipping Box<str> until [this PR](https://github.com/paritytech/parity-scale-codec/pull/565) is merged
+        let exceptions: [core::any::TypeId; 1] = [core::any::TypeId::of::<Box<str>>()];
```

Comment:
Have you considered using static memory instead of heap for `ChainId` ? Not compatible with wasm?

Reply:
good point, we might try doing it

Reply:
opened #4336